lib/iris/tests/experimental/regrid/test_regrid_area_weighted_rectilinear_src_and_grid.py

```diff
@@ -495,39 +466,35 @@ def test_scalar_source_cube(self):
         self.assertEqual(res.data, src.data)
 
     @tests.skip_data
-    @tests.skip_plot
     def test_global_data_reduce_res(self):
         src = iris.tests.stock.global_pp()
         src.coord('latitude').guess_bounds()
         src.coord('longitude').guess_bounds()
         dest = _resampled_grid(src, 0.4, 0.3)
         res = regrid_area_weighted(src, dest)
-        qplt.pcolormesh(res)
-        self.check_graphic()
+        self.assertArrayAlmostEqual(res.data.mean(), 280.137698)
```

Comment:
> Out of interest, why have you used 'AlmostEqual', rather than 'Equal'?

because every time someone compares two floats for equality in a test, a fairy dies ;)